server/src/internalClusterTest/java/org/opensearch/remotemigration/RemotePrimaryRelocationIT.java

```diff
@@ -86,7 +101,13 @@ public void testRemotePrimaryRelocation() throws Exception {
         // Index some more docs
         int currentDoc = numAutoGenDocs.get();
         int finalCurrentDoc1 = currentDoc;
-        waitUntil(() -> numAutoGenDocs.get() > finalCurrentDoc1 + 5);
+        waitUntil(() -> numAutoGenDocs.get() > finalCurrentDoc1 + 100);
+
+        ByteSizeValue shardSize = client().admin().indices().prepareStats(INDEX_NAME).execute().actionGet().getShards()[0].getStats()
+            .getStore()
+            .size();
+        logger.info("Shard size after migration is {}", shardSize);
+        slowDownRecovery(shardSize);
```

Comment:
is this part of the fix or just an improvement in the test to bring it closed to real world scenario?

Reply:
This is just an improvement to make it closer to real world scenario